src/Bathymetry.jl

```diff
@@ -126,6 +124,14 @@ function regrid_bathymetry(target_grid;
                            interpolation_passes = 1,
```

Comment:
```suggestion
                           smoothing = InterpolationPasses(0),
```

Reply:
ok, you are suggesting we do the refactor of the bathymetry in this PR?